### DISCUSSION TITLE
text box change color of text

### DISCUSSION THREAD

**Discussion by PercyWoeds** [prawnpdf/prawn#1245] [General] [answered]
Hello

I need to change the color of text in a text box how can i do this ?

Thanks

**Comment by pointlessone** [comment, marked as answer]
You need to set appropriate fill for your text.

```ruby
fill_color 'ff0000'
text "Hello world!"
```

You can also pass color along with the text.

```ruby
test "Hello world", color: "ff0000"
```

I believe both of these and a few more covered in [the manual](https://prawnpdf.org/manual.pdf).
